sycl/include/sycl/ext/oneapi/matrix/matrix.hpp

```diff
@@ -17,14 +17,13 @@
 #include <CL/sycl/feature_test.hpp>
 
 #if (SYCL_EXT_ONEAPI_MATRIX == 1)
```

Comment:
add a comment that the default is matrix-jit-use and that existing tests like in llvm-test-suite will not fail because we have the "unnecessary" use value